app/gui/src/project-view/util/__tests__/callTree.test.ts

```diff
@@ -52,7 +51,6 @@ const infixFixture = {
       'lhs',
       { kind: 'Multiple_Choice', display: widgetCfg.DisplayMode.Always, label: null, values: [] },
     ],
-    ['rhs', { kind: 'Code_Input', display: widgetCfg.DisplayMode.Always }],
```

Comment:
Haven't we stopped covering some path? Like, assigning dynamic configuration to operator's right argument?